infrastructure/demo/main/gh.tf

```diff
@@ -0,0 +1,105 @@
+# Github OICD provider
+data "aws_iam_openid_connect_provider" "github" {
+  url = "https://token.actions.githubusercontent.com"
+}
+
+data "aws_iam_policy_document" "github_actions_assume_role" {
+  statement {
+    actions = ["sts:AssumeRoleWithWebIdentity"]
+    principals {
+      type        = "Federated"
+      identifiers = [data.aws_iam_openid_connect_provider.github.arn]
+    }
+    condition {
+      test     = "StringEquals"
+      variable = "token.actions.githubusercontent.com:aud"
+      values   = ["sts.amazonaws.com"]
+    }
+    condition {
+      test     = "StringLike"
+      variable = "token.actions.githubusercontent.com:sub"
+      values = [
+        "repo:IDinsight/aaq-core:*"
```

Comment:
If I'm creating a new project, I'd need to change this value right?
Should we just keep a project-level tfvar file where we define project-level variables like this, that both tf_backend and `demo` and other environments can use?

Off the top of my head these are
```
project_name
billing_code
repository
aws_region
```

It's ok to keep this as a backlog ticket though!

Reply:
Yes this makes sense.